ui/apps/dashboard/src/components/Metrics/utils.ts

```diff
@@ -79,13 +76,44 @@ export const sum = (data?: MetricsData[]) =>
 
 export const formatNumber = (number?: number | bigint) => (number || 0).toLocaleString(undefined);
 
+export const marker = (color: string) =>
+  `<span style="display:inline-block;margin-right:4px;border-radius:10px;width:10px;height:10px;
+      border-width: 1px;border-color:${color};background-color:${color};"></span>`;
+
+export const formatDimension = (param: any) => {
+  const color = typeof param.color === 'object' ? param.color?.colorStops[0]?.color : param.color;
+
+  //
+  // FYI using vanilla html in formatter because rendering react here causes
+  // some lag with synched cursors
+  return `<div class="flex flex-row justify-between flex-nowrap items-center px-2">
+    <div>
+      <span class="mr">${marker(color)}
```

Comment:
Nitpicking: The elements without children are more readable with self closing tags